Source/API/CBLReplication.m

```diff
@@ -387,6 +387,21 @@ - (void) updateStatus: (CBLReplicationStatus)status
     }
 }
 
+- (void)forgetReplication {
```

Comment:
So because of the above two notes, I don't think this call to the bg thread is needed. Or did I overlook something?